lib/open_project/journal_formatter/attachment.rb

```diff
@@ -27,8 +27,6 @@
 #++
 
 class OpenProject::JournalFormatter::Attachment < ::JournalFormatter::Base
-  # unloadable
```

Comment:
There are quite a few of those in here @myabc. Can you please get rid of them for good?